### DISCUSSION TITLE
spaCy consulting & jobs

### DISCUSSION THREAD

**Discussion by ines** [explosion/spaCy#10259] [News & Announcements]
This thread is a space for companies and developers to connect about job opportunities and consulting engagements related to spaCy and NLP ✨

## <a href="https://explosion.ai/spacy-tailored-pipelines"><img width="350" alt="spacy-tailored-pipelines_wide" src="https://user-images.githubusercontent.com/13643239/153482413-0d9d9c2c-033a-4778-8b4c-0d845a67b833.png" alt="spaCy Tailored Pipelines"></a>

If you need help with building applied NLP pipelines with spaCy and jumpstarting your NLP project, check out our new offering, [spaCy Tailored Pipelines](https://explosion.ai/spacy-tailored-pipelines). Get a custom spaCy pipeline, tailor-made for your NLP problem by spaCy's core developers. Streamlined, production-ready, predictable and maintainable, with up-front pricing and try before you buy. [**Learn more →**](https://explosion.ai/spacy-tailored-pipelines)

## If you're hiring or looking for consultants

Feel free to post your job ad or consulting inquiry on this thread and include some details on the project and the level of spaCy expertise you're looking for. Don't forget to include contact details so people can reach out. Others can also comment on your post with any questions.

## If you're looking for work

Feel free to post on the thread and include details of what you're looking for (full-time position, consulting engagement), as well as your previous experience and links to your portfolio. Don't forget to include contact details so companies can reach out. Others can also comment on your post with any questions.

- [spaCy jobs on Indeed](https://www.indeed.com/jobs?q=spacy)
- [spaCy jobs on Monster](https://www.monster.com/jobs/search?q=spacy)
- [spaCy jobs on LinkedIn](https://www.linkedin.com/jobs/search/?keywords=spacy)

**Comment by tdubon** [comment]
Hi, 
I can be reached at tdubon6@gmail.com for full-time positions or consulting engagements. I have a lot of experience working one on one with stakeholders and delivering on projects. Here are links to some of my work. 

Github: [github.com/tdubon](https://github.com/tdubon)
Medium: [medium.com/@tdubon6](https://medium.com/@tdubon6) 
LinkedIn: [linkedin.com/in/tanniadubon](https://www.linkedin.com/in/tanniadubon)
Hugging Face: [huggingface.co/spaces/tdubon/CustomizeNER](https://huggingface.co/spaces/tdubon/CustomizeNER), [huggingface.co/keras-io/TF_Decision_Trees](https://huggingface.co/keras-io/TF_Decision_Trees)
naas: [jupyter-naas/awesome-notebooks/Twitter_Get_tweets_stats_from_profile](https://github.com/jupyter-naas/awesome-notebooks/blob/master/Twitter/Twitter_Get_tweets_stats_from_profile.ipynb)
[Dubont_Resume.pdf](https://github.com/explosion/spaCy/files/8103239/Dubont_Resume.pdf)

**Comment by ivyleavedtoadflax** [comment]
Hi 👋

We are Mantis, a remote, EU based consultancy specializing in NLP. We have experience working with tech startups, public sector, nonprofit, and enterprise organizations. We believe in open source software, practical solutions, and trustworthy AI 🤖. 

We are heavy users of spaCy and Prodigy, and have experience writing our own spaCy pipelines and Prodigy recipes. 😎.  We have used spaCy and Prodigy to:

* Extract entities (e.g.  organizations) from policy documents, and locations from grant applications (NER).
* Identify conditions and diseases in the biomedical domain (Extreme Multilabel classification) [[link](https://medium.com/wellcome-data/tagging-biomedical-grants-with-29k-tags-b18273b3cbf9)]
* Annotate bibliographic references to allow us to train a BiLSTM to extract them from policy documents (token classification) [[link](https://github.com/wellcometrust/deep_reference_parser)]
* Many more… [See our case studies here](https://mantisnlp.com/work/)

We also work extensively with transformers, RASA, and other popular frameworks.

Reach out at hi@mantisnlp.com if you are interested in doing a project with us 🚀.

Find out about us here:

- [Mantis Website](https://www.mantisnlp.com)
- [Blog](https://medium.com/mantisnlp)
- [LinkedIn](https://www.linkedin.com/company/mantisnlp)
- [Github](https://github.com/MantisAI)

**Comment by luvogels** [comment]
Hi,

Upfeed in Oslo, Norway are hiring NLP engineers with spaCy experience:

[Linkedin](https://www.linkedin.com/jobs/view/2965888897/?refId=8PU0SBJ4TUqp%2BR7Jtdzpwg%3D%3D&lipi=urn%3Ali%3Apage%3Ad_flagship3_hiring_applicant_index%3BKM4JQvCMQtK4zi%2F2HELGCA%3D%3D&licu=urn%3Ali%3Acontrol%3Ad_flagship3_hiring_applicant_index-object&lici=aYCDP%2B%2BhQcy4nwn9ekESyA%3D%3D)

**Comment by nrodnova** [comment]
We are hiring a Senior Data Scientist/Computational Linguist with experience in Healthcare domain. EON is a mid-stage startup, super-friendly and with a great mission.

Here's the info about the company:
https://eonhealth.com/about/

And here's the job post: [Senior Data Scientist](https://www.linkedin.com/jobs/view/2978472330/?refId=vDjOY714QgK%2Fu0UEJfy2TA%3D%3D)
